Description: Update certificates to use 2048 bits key
 The small key size was causing a FTBFS in Ubuntu, where this error has been
 reported: SSL_CTX_use_certificate:ee key too small.
Forwarded: no
Author: Lucas Kanashiro <lucas.kanashiro@canonical.com>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2020-04-03

--- a/t/SSL_test.t
+++ b/t/SSL_test.t
@@ -16,34 +16,88 @@
 
 my $pem = << 'PEM'; # this certificate is invalid, please only use for testing
 -----BEGIN CERTIFICATE-----
-MIICKTCCAZICCQDFxHnOjdmTTjANBgkqhkiG9w0BAQUFADBZMQswCQYDVQQGEwJB
-VTETMBEGA1UECAwKU29tZS1TdGF0ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lkZ2l0
-cyBQdHkgTHRkMRIwEAYDVQQDDAlsb2NhbGhvc3QwHhcNMTIwMTE0MTgzMjMwWhcN
-NzUxMTE0MTIwNDE0WjBZMQswCQYDVQQGEwJBVTETMBEGA1UECAwKU29tZS1TdGF0
-ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lkZ2l0cyBQdHkgTHRkMRIwEAYDVQQDDAls
-b2NhbGhvc3QwgZ8wDQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBAKLGfQantHdi/0cd
-eoOHRbWKChpI/g84hU8SnwmrSMZR0x76vDLKMDYohISoKxRPx6j2M2x3P4K+kEJm
-C5H9iGdD9p9ljGnRdkGp5yYeuwWfePRb4AOwP5qgQtEb0OctFIMjcAIIAw/lsnUs
-hGnom0+uA9W2H63PgO0o4qiVAn7NAgMBAAEwDQYJKoZIhvcNAQEFBQADgYEATDGA
-dYRl5wpsYcpLgNzu0M4SENV0DAE2wNTZ4LIR1wxHbcxdgzMhjp0wwfVQBTJFNqWu
-DbeIFt4ghPMsUQKmMc4+og2Zyll8qev8oNgWQneKjDAEKKpzdvUoRZyGx1ZocGzi
-S4LDiMd4qhD+GGePcHwmR8x/okoq58xZO/+Qygc=
+MIIFazCCA1OgAwIBAgIUA8Xm/EUFCN3yY1jqQqfivcJPAxcwDQYJKoZIhvcNAQEL
+BQAwRTELMAkGA1UEBhMCQVUxEzARBgNVBAgMClNvbWUtU3RhdGUxITAfBgNVBAoM
+GEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDAeFw0yMDA0MDExMzA5MDRaFw00NzA4
+MTgxMzA5MDRaMEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEw
+HwYDVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwggIiMA0GCSqGSIb3DQEB
+AQUAA4ICDwAwggIKAoICAQDH7zYyow2nvRQeqa8dPtJCi65OcDvWUmL4Fazxas56
+kYHv3gdEAVcxrW1AFhziDNvrsLiWE9WfFoav9xEPtXvDsY9+2rsNoiTUzPgvYaFG
+5Uz43sz4gqxbndxWTtMbpFA0zrhXnAHm4EUM2ykU04KQXHMGFTAHb+c7ARKMg7+B
+H6j4XiSEZenWRqh9bE71wIMs2gvsRuVzTZrobzYHSrHJYyhjBoEmWahm9PFAfkvF
+tZl9hmaz0jPYDmHdzqusMT3lNvZucGn76Z+8KCFmnMb8lGmfMBbumHpjiJ53DLGK
+KEhR7kK/86t9lnAdvcIPlDTJPk/xHoxXhbHcKtEnnKat1a3/THSTrGT96OYlpjm/
+JwjbLoFGjT9D06oajWY/lt/CeAQy76qRHqJVJyZ2j9TXw3tEQvawTxPMPIE+iKfs
+78OG4d/j9M1R4tkPQdwD8VCVJB7e6+2HlbPai/djMUqkj2stMZ3sgv5ehHae0xth
+BM29I/TdOLfgP8S3EGqVVYcyRAWKYvGYwgGEvocmiUgXjowOsOt1h5a+CD/Qfodg
+6qfilPpD2aZYqcPSn5Htqp+pkjMpWC1aMflxQbxvXcJFTzGbc4HslqJhbLXe/fwR
+2scOg8ZEt8Y94sF/7Y+GLDaJnV8ObmdMttzx7HWdQMtPkFvo4/h7fseG/bA4/SoX
+xQIDAQABo1MwUTAdBgNVHQ4EFgQUntWE93uYLQ+bbKqFswQyG0aqZkowHwYDVR0j
+BBgwFoAUntWE93uYLQ+bbKqFswQyG0aqZkowDwYDVR0TAQH/BAUwAwEB/zANBgkq
+hkiG9w0BAQsFAAOCAgEAt2+PAKmobIRb4+5vGgfhLzPW97yCi03szpfe9mUwmeFN
+EcLXRl1t2lyLc/Ucn8pSUTfEv1WT96JYbTurUkM1iLi+y5jJeS8qAA59Me1HcPvF
+vMy7MG/Fam0wU/wEC0wzWwPDDIUG5PM9rk9vkBmZ44TltE//i2wbh8Zo7z1nNUDy
+ms6K8pQjoG7SJefHbCjyqYrm17pb2/ClIZuZWs8rvot/9zslKiDKNK4ewdY0iONy
+q861PZ+TqdTpxm8ouBkpQA2ggIZNcfwO/KVr6nqBVp072dXlaSRiBD+z4CNnGb4b
+Gz931Iev0zTKY3m9uL8jNO36BRB4paIaDexeYxK01L2mFHkCZOukhYOB4qli8+4y
+/vlOaMuuhVeQNdjjq2t1k1wP1+1QjHdyimenDYFnvzTnu7hBr5Wgs0/sxCBug9aJ
+/v6rW/kItPbLmgoo6Q9sJWEzJBjUzacaus/7HYa2XMQ6qq+dP+HMcaCtDCzbKgyJ
+w1EMMd4f/uZeinE5BDljhSuJLl6vVP0WyBR3CYQdbpQlc1Koansr/j97OyZtwTmY
+6+xIh1WATcvrzSVJiLJb9zTcOg8SXtAHpAkpEGQk1kDiJUP+3AN2uOT3KrlxSjCg
+skbx0I2wRdK4UvTr3WAakSjdmxMoHzAyoZ1OqBiz9ndGWWY8i5acAlcgFIfa730=
 -----END CERTIFICATE-----
------BEGIN RSA PRIVATE KEY-----
-MIICXAIBAAKBgQCixn0Gp7R3Yv9HHXqDh0W1igoaSP4POIVPEp8Jq0jGUdMe+rwy
-yjA2KISEqCsUT8eo9jNsdz+CvpBCZguR/YhnQ/afZYxp0XZBqecmHrsFn3j0W+AD
-sD+aoELRG9DnLRSDI3ACCAMP5bJ1LIRp6JtPrgPVth+tz4DtKOKolQJ+zQIDAQAB
-AoGASXDmvhbyfJ8k8HAjc66XzBWxAzUFs9Zbh1aufM1UM259o8+bFAtXf0f+ql+5
-uBtaySf0Aa8374SNT/f8pmzOmpiXMvYRz8Z5Gc6JYpYd/PrCoSCGtP+NdCvk7Y5c
-eUmmpiEto4+fgCAKrtqc5jm8eBWn/yNhQNDBVJ9qX+kXQOECQQDVBLvBZaECSMTm
-djKuPlZ93cmyI7g+TURTl2N08fz4xQVVbo5+AV0GsEZupBpTgrHpLTk8gKP/nfdR
-9KWZldbZAkEAw55+SqrVTv4cI0fMvC0t8Wl46zTkY9tK65TGnbO1DbTQh9qs+NwH
-+v3uu47ef5w/73xLtDjQouz//0z5rgF3FQJAfrmOKQOYwY8g9CmlBNu5ALAM6Zku
-ZoH4//G0DUJYyHYNMkHPK08MVIpRnEisELpTtPBeeIvfBJapJ2xvh+sIIQJASeY4
-I5EB4EOS8akQKQ6QSqDjs0dZ+HdBiFm95pmbDkB+frQXoDPPN/xyEZzZZS/r31b/
-amgEOWh7FUFJGXkoOQJBALfOgsiss0lASlOXAg1rwO4m2OaDiaEde01PLcSjIaKl
-Qfbzc7ZYF+fGDsHHlD5Kgj1CGaWCVVHqCv4UHSrA/gM=
------END RSA PRIVATE KEY-----
+-----BEGIN PRIVATE KEY-----
+MIIJQwIBADANBgkqhkiG9w0BAQEFAASCCS0wggkpAgEAAoICAQDH7zYyow2nvRQe
+qa8dPtJCi65OcDvWUmL4Fazxas56kYHv3gdEAVcxrW1AFhziDNvrsLiWE9WfFoav
+9xEPtXvDsY9+2rsNoiTUzPgvYaFG5Uz43sz4gqxbndxWTtMbpFA0zrhXnAHm4EUM
+2ykU04KQXHMGFTAHb+c7ARKMg7+BH6j4XiSEZenWRqh9bE71wIMs2gvsRuVzTZro
+bzYHSrHJYyhjBoEmWahm9PFAfkvFtZl9hmaz0jPYDmHdzqusMT3lNvZucGn76Z+8
+KCFmnMb8lGmfMBbumHpjiJ53DLGKKEhR7kK/86t9lnAdvcIPlDTJPk/xHoxXhbHc
+KtEnnKat1a3/THSTrGT96OYlpjm/JwjbLoFGjT9D06oajWY/lt/CeAQy76qRHqJV
+JyZ2j9TXw3tEQvawTxPMPIE+iKfs78OG4d/j9M1R4tkPQdwD8VCVJB7e6+2HlbPa
+i/djMUqkj2stMZ3sgv5ehHae0xthBM29I/TdOLfgP8S3EGqVVYcyRAWKYvGYwgGE
+vocmiUgXjowOsOt1h5a+CD/Qfodg6qfilPpD2aZYqcPSn5Htqp+pkjMpWC1aMflx
+QbxvXcJFTzGbc4HslqJhbLXe/fwR2scOg8ZEt8Y94sF/7Y+GLDaJnV8ObmdMttzx
+7HWdQMtPkFvo4/h7fseG/bA4/SoXxQIDAQABAoICAAbYhf6N3rXTn5C9NqXFtOVa
+awl8hk/8Wi8sbtOFWLSRruVLsOv/L8EfsxHyr+J9ljonvupEm5fq6Ym05/yltisp
+NUSesLDy0FgI/KaCrUcEKvKKjnIj50rryNObt1bG9YgZW+6EBPymyTZ7epif9WSE
+Bdw7dX2Ls1st2ji9eh0tvFdkwdNWuf8ARFynDL0VbmhmvunEM68TBS7YP/1X7WZ7
+4rIhUuLBRybfVDNlH6sRYMQPigy2MdhABdHWdcJbnUbv7tgxOS/K/BExPpOI4rdb
+TZKJzv80cVxfHS3uXVXhszg69EYmTcTrFcOu76og5P3PCGW1KhEFHuXvAWJd1scl
+cPPztRlnE3/2gpedbnl2X6bGBjRRnW8qT5A253I03fHT9wYJuoTkoz4wjLAZjojG
+ytjRn0ZN9zWfBIb1Tz2M5uiIepye9hZrRNWWegAJlUkHyJ4xMTx3A7m2ZPDj+JqD
+01rXZ0MmEJTs1Y0LGzGFT55GzZVUJDWunrNGmhoZkWg6TJpfF73nIHUJNjK8qPTo
+Q3Y2r2eHGZvbzdd+mhd+i+1ol8CW3+yqBge6EWDkYR/01rBfoNdiJBWkzT/qnQvi
+UIvsC6I++kXK0KIhtO7+5q9vpx8IZBtggFUtRvNaYRrvunalW4rSyxzHd6lX7N9F
+C8P1on/atUxUDlv3gF8xAoIBAQD2lbpwQPAzAYhjowowdmzTN0wKba5RTLj++Wvp
+is//gliX/MauUBdXzkVxWgpODr4znsuAH8eQXKADw1wI1iCozBbapQp934r3c6JJ
+LwFRKRUalYWMwckp7IAo7k7Vu2u8Y8k/T61uKLWdgIClo2zYBYpfdYpCOPV6SIMF
+L1dS5M8pac9CkM2NbcEMbZcXykjZmrZidIrDkI3kTPqUjSN+pguUxoJg+I+gUEzQ
+FQj1VJR5SViJk49GMDs0vo+frOAW2kc2RdvvPnuPpplyEySjcMqZtx+vDmVsRAjK
+AFXaPSy5liGCg9j2N9Ab9R6w9JkB3QXM4XFBvQHar6rdP4l/AoIBAQDPkX5IO0T+
+iIgALd+tpyfKywoohzwTlzAM9qsqDmr+BX28Vphu7DOZLhd9zFlKmZHhlj3XXRP0
+Y7Y8JbO+ir2okYNDkA6s7EAl5C/m7RDRvj8d4huPm1rptBq3J0dcwgwOI2Dk1QdC
+CrWEkYLzfObiRUd7gDjBYt9/KsK52Q04iYphs2ItXtrZf6rdFQC+MIJcX35ko9dQ
+cYs+rY3DdwjfEVwwcMTShE/jRMcN4PKnX5QwVzIyZeJPU4DJlNYs3Iaa292GwQ0p
+YnDrYqv0Dy2Tx5/TaEdQdbRX8wGtz+pZresrRMacPN8t8WM7O38Qug5ouwH19DBu
+pjzh5aEUA1i7AoIBABE9aqmKgMCwLL76mS3GOdmSlihsfrGEcbKx8Y+EewJcNKF7
+tNBfHSKwcz53kxzd/wJQ6d1tW2CGeVGKCRc9EU975WUoANHIHUkrtn7zYF4yRx1y
+ssGiktPxiwxRjQV4cxHa0CkzAucexYPbhiMOh/+ac5A1AZObs932z+I+6xYKlUlJ
+8omu4hAvSj36M4QgSnOcU4ASsdj2dFUv5J0aOQ8TwN+H+XmaJ0CIHLa3oca1QSQx
+spT70hqQKLOJVzVMuuYeILh0renOLoleln/ZQsiCjEeu+/IbSZAGOa8V0urNOCFJ
+k9IyMasVP+GUg67PixsMPumSIX79HfISMhoB5TUCggEBAMVULBm/Pvg8FA8XjW4p
+W0sPe7jL1/FH6gZo+pAg5NZZog9Kw9+v7d3SU8LkYn7pQCaWDnSPqEjOApFrxlV+
+0I9QxtmUOl9quhFLvb5r4XGEy7w9GLaNmwBSmJNGZDFqyMsoFxV08FF4nNhK/ZM9
+SsIR2sMuQsaWmKLso/LKxibZmxUG1G8NnkDnfihvryUgOM5YenBy0l9HknkjxYHt
+yCFI/7uNeZAo+Um2OQaYtBcqZlcOjkobUerYF7eMJ5C+lbjjDNbu8PRHAdLFG3QK
+eenj/a2dlS6It8pk21PCNajMDqYz3BzsQcALm6rUBRiByPEH1/VbEDAhGgAnrdq4
+08ECggEBAIO+7DW3vfitCetSYk5oJAVNN+9OMs4SR8/2cbuMc9a/Chc6O/rj58EP
+yH4wSifpnesoZzCah3Ryy+RTYXEwNV65Xopd+J/ANt9MjP8c4j0hY3n/BzpI4aN3
+dpQyXWBbsECZAl0B0LKA2mLMAo72SQYXK9F62Zl6LVoT27wfsADYTqM9GcFagp6/
+fzKQWdLgRps34ysE8PSYkXaIfq/Q0uv1xvDMdW5GpNxcfkikZXQJnuUlxKvdf7Zi
+bdoZvJJ6MPNNoCZBOM+WSitRculiJHfeEMU5VQXpXJby0YEt+Gc4q3Zs1tv3aOqF
+k+WUbp8EAMRqFNRRLcPzcux5vlLvOtw=
+-----END PRIVATE KEY-----
 PEM
 
 my ($pem_fh, $pem_filename) =
--- a/t/SSLEAY_test.t
+++ b/t/SSLEAY_test.t
@@ -21,34 +21,88 @@
 
 my $pem = << 'PEM'; # this certificate is invalid, please only use for testing
 -----BEGIN CERTIFICATE-----
-MIICKTCCAZICCQDFxHnOjdmTTjANBgkqhkiG9w0BAQUFADBZMQswCQYDVQQGEwJB
-VTETMBEGA1UECAwKU29tZS1TdGF0ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lkZ2l0
-cyBQdHkgTHRkMRIwEAYDVQQDDAlsb2NhbGhvc3QwHhcNMTIwMTE0MTgzMjMwWhcN
-NzUxMTE0MTIwNDE0WjBZMQswCQYDVQQGEwJBVTETMBEGA1UECAwKU29tZS1TdGF0
-ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lkZ2l0cyBQdHkgTHRkMRIwEAYDVQQDDAls
-b2NhbGhvc3QwgZ8wDQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBAKLGfQantHdi/0cd
-eoOHRbWKChpI/g84hU8SnwmrSMZR0x76vDLKMDYohISoKxRPx6j2M2x3P4K+kEJm
-C5H9iGdD9p9ljGnRdkGp5yYeuwWfePRb4AOwP5qgQtEb0OctFIMjcAIIAw/lsnUs
-hGnom0+uA9W2H63PgO0o4qiVAn7NAgMBAAEwDQYJKoZIhvcNAQEFBQADgYEATDGA
-dYRl5wpsYcpLgNzu0M4SENV0DAE2wNTZ4LIR1wxHbcxdgzMhjp0wwfVQBTJFNqWu
-DbeIFt4ghPMsUQKmMc4+og2Zyll8qev8oNgWQneKjDAEKKpzdvUoRZyGx1ZocGzi
-S4LDiMd4qhD+GGePcHwmR8x/okoq58xZO/+Qygc=
+MIIFazCCA1OgAwIBAgIUA8Xm/EUFCN3yY1jqQqfivcJPAxcwDQYJKoZIhvcNAQEL
+BQAwRTELMAkGA1UEBhMCQVUxEzARBgNVBAgMClNvbWUtU3RhdGUxITAfBgNVBAoM
+GEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDAeFw0yMDA0MDExMzA5MDRaFw00NzA4
+MTgxMzA5MDRaMEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEw
+HwYDVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwggIiMA0GCSqGSIb3DQEB
+AQUAA4ICDwAwggIKAoICAQDH7zYyow2nvRQeqa8dPtJCi65OcDvWUmL4Fazxas56
+kYHv3gdEAVcxrW1AFhziDNvrsLiWE9WfFoav9xEPtXvDsY9+2rsNoiTUzPgvYaFG
+5Uz43sz4gqxbndxWTtMbpFA0zrhXnAHm4EUM2ykU04KQXHMGFTAHb+c7ARKMg7+B
+H6j4XiSEZenWRqh9bE71wIMs2gvsRuVzTZrobzYHSrHJYyhjBoEmWahm9PFAfkvF
+tZl9hmaz0jPYDmHdzqusMT3lNvZucGn76Z+8KCFmnMb8lGmfMBbumHpjiJ53DLGK
+KEhR7kK/86t9lnAdvcIPlDTJPk/xHoxXhbHcKtEnnKat1a3/THSTrGT96OYlpjm/
+JwjbLoFGjT9D06oajWY/lt/CeAQy76qRHqJVJyZ2j9TXw3tEQvawTxPMPIE+iKfs
+78OG4d/j9M1R4tkPQdwD8VCVJB7e6+2HlbPai/djMUqkj2stMZ3sgv5ehHae0xth
+BM29I/TdOLfgP8S3EGqVVYcyRAWKYvGYwgGEvocmiUgXjowOsOt1h5a+CD/Qfodg
+6qfilPpD2aZYqcPSn5Htqp+pkjMpWC1aMflxQbxvXcJFTzGbc4HslqJhbLXe/fwR
+2scOg8ZEt8Y94sF/7Y+GLDaJnV8ObmdMttzx7HWdQMtPkFvo4/h7fseG/bA4/SoX
+xQIDAQABo1MwUTAdBgNVHQ4EFgQUntWE93uYLQ+bbKqFswQyG0aqZkowHwYDVR0j
+BBgwFoAUntWE93uYLQ+bbKqFswQyG0aqZkowDwYDVR0TAQH/BAUwAwEB/zANBgkq
+hkiG9w0BAQsFAAOCAgEAt2+PAKmobIRb4+5vGgfhLzPW97yCi03szpfe9mUwmeFN
+EcLXRl1t2lyLc/Ucn8pSUTfEv1WT96JYbTurUkM1iLi+y5jJeS8qAA59Me1HcPvF
+vMy7MG/Fam0wU/wEC0wzWwPDDIUG5PM9rk9vkBmZ44TltE//i2wbh8Zo7z1nNUDy
+ms6K8pQjoG7SJefHbCjyqYrm17pb2/ClIZuZWs8rvot/9zslKiDKNK4ewdY0iONy
+q861PZ+TqdTpxm8ouBkpQA2ggIZNcfwO/KVr6nqBVp072dXlaSRiBD+z4CNnGb4b
+Gz931Iev0zTKY3m9uL8jNO36BRB4paIaDexeYxK01L2mFHkCZOukhYOB4qli8+4y
+/vlOaMuuhVeQNdjjq2t1k1wP1+1QjHdyimenDYFnvzTnu7hBr5Wgs0/sxCBug9aJ
+/v6rW/kItPbLmgoo6Q9sJWEzJBjUzacaus/7HYa2XMQ6qq+dP+HMcaCtDCzbKgyJ
+w1EMMd4f/uZeinE5BDljhSuJLl6vVP0WyBR3CYQdbpQlc1Koansr/j97OyZtwTmY
+6+xIh1WATcvrzSVJiLJb9zTcOg8SXtAHpAkpEGQk1kDiJUP+3AN2uOT3KrlxSjCg
+skbx0I2wRdK4UvTr3WAakSjdmxMoHzAyoZ1OqBiz9ndGWWY8i5acAlcgFIfa730=
 -----END CERTIFICATE-----
------BEGIN RSA PRIVATE KEY-----
-MIICXAIBAAKBgQCixn0Gp7R3Yv9HHXqDh0W1igoaSP4POIVPEp8Jq0jGUdMe+rwy
-yjA2KISEqCsUT8eo9jNsdz+CvpBCZguR/YhnQ/afZYxp0XZBqecmHrsFn3j0W+AD
-sD+aoELRG9DnLRSDI3ACCAMP5bJ1LIRp6JtPrgPVth+tz4DtKOKolQJ+zQIDAQAB
-AoGASXDmvhbyfJ8k8HAjc66XzBWxAzUFs9Zbh1aufM1UM259o8+bFAtXf0f+ql+5
-uBtaySf0Aa8374SNT/f8pmzOmpiXMvYRz8Z5Gc6JYpYd/PrCoSCGtP+NdCvk7Y5c
-eUmmpiEto4+fgCAKrtqc5jm8eBWn/yNhQNDBVJ9qX+kXQOECQQDVBLvBZaECSMTm
-djKuPlZ93cmyI7g+TURTl2N08fz4xQVVbo5+AV0GsEZupBpTgrHpLTk8gKP/nfdR
-9KWZldbZAkEAw55+SqrVTv4cI0fMvC0t8Wl46zTkY9tK65TGnbO1DbTQh9qs+NwH
-+v3uu47ef5w/73xLtDjQouz//0z5rgF3FQJAfrmOKQOYwY8g9CmlBNu5ALAM6Zku
-ZoH4//G0DUJYyHYNMkHPK08MVIpRnEisELpTtPBeeIvfBJapJ2xvh+sIIQJASeY4
-I5EB4EOS8akQKQ6QSqDjs0dZ+HdBiFm95pmbDkB+frQXoDPPN/xyEZzZZS/r31b/
-amgEOWh7FUFJGXkoOQJBALfOgsiss0lASlOXAg1rwO4m2OaDiaEde01PLcSjIaKl
-Qfbzc7ZYF+fGDsHHlD5Kgj1CGaWCVVHqCv4UHSrA/gM=
------END RSA PRIVATE KEY-----
+-----BEGIN PRIVATE KEY-----
+MIIJQwIBADANBgkqhkiG9w0BAQEFAASCCS0wggkpAgEAAoICAQDH7zYyow2nvRQe
+qa8dPtJCi65OcDvWUmL4Fazxas56kYHv3gdEAVcxrW1AFhziDNvrsLiWE9WfFoav
+9xEPtXvDsY9+2rsNoiTUzPgvYaFG5Uz43sz4gqxbndxWTtMbpFA0zrhXnAHm4EUM
+2ykU04KQXHMGFTAHb+c7ARKMg7+BH6j4XiSEZenWRqh9bE71wIMs2gvsRuVzTZro
+bzYHSrHJYyhjBoEmWahm9PFAfkvFtZl9hmaz0jPYDmHdzqusMT3lNvZucGn76Z+8
+KCFmnMb8lGmfMBbumHpjiJ53DLGKKEhR7kK/86t9lnAdvcIPlDTJPk/xHoxXhbHc
+KtEnnKat1a3/THSTrGT96OYlpjm/JwjbLoFGjT9D06oajWY/lt/CeAQy76qRHqJV
+JyZ2j9TXw3tEQvawTxPMPIE+iKfs78OG4d/j9M1R4tkPQdwD8VCVJB7e6+2HlbPa
+i/djMUqkj2stMZ3sgv5ehHae0xthBM29I/TdOLfgP8S3EGqVVYcyRAWKYvGYwgGE
+vocmiUgXjowOsOt1h5a+CD/Qfodg6qfilPpD2aZYqcPSn5Htqp+pkjMpWC1aMflx
+QbxvXcJFTzGbc4HslqJhbLXe/fwR2scOg8ZEt8Y94sF/7Y+GLDaJnV8ObmdMttzx
+7HWdQMtPkFvo4/h7fseG/bA4/SoXxQIDAQABAoICAAbYhf6N3rXTn5C9NqXFtOVa
+awl8hk/8Wi8sbtOFWLSRruVLsOv/L8EfsxHyr+J9ljonvupEm5fq6Ym05/yltisp
+NUSesLDy0FgI/KaCrUcEKvKKjnIj50rryNObt1bG9YgZW+6EBPymyTZ7epif9WSE
+Bdw7dX2Ls1st2ji9eh0tvFdkwdNWuf8ARFynDL0VbmhmvunEM68TBS7YP/1X7WZ7
+4rIhUuLBRybfVDNlH6sRYMQPigy2MdhABdHWdcJbnUbv7tgxOS/K/BExPpOI4rdb
+TZKJzv80cVxfHS3uXVXhszg69EYmTcTrFcOu76og5P3PCGW1KhEFHuXvAWJd1scl
+cPPztRlnE3/2gpedbnl2X6bGBjRRnW8qT5A253I03fHT9wYJuoTkoz4wjLAZjojG
+ytjRn0ZN9zWfBIb1Tz2M5uiIepye9hZrRNWWegAJlUkHyJ4xMTx3A7m2ZPDj+JqD
+01rXZ0MmEJTs1Y0LGzGFT55GzZVUJDWunrNGmhoZkWg6TJpfF73nIHUJNjK8qPTo
+Q3Y2r2eHGZvbzdd+mhd+i+1ol8CW3+yqBge6EWDkYR/01rBfoNdiJBWkzT/qnQvi
+UIvsC6I++kXK0KIhtO7+5q9vpx8IZBtggFUtRvNaYRrvunalW4rSyxzHd6lX7N9F
+C8P1on/atUxUDlv3gF8xAoIBAQD2lbpwQPAzAYhjowowdmzTN0wKba5RTLj++Wvp
+is//gliX/MauUBdXzkVxWgpODr4znsuAH8eQXKADw1wI1iCozBbapQp934r3c6JJ
+LwFRKRUalYWMwckp7IAo7k7Vu2u8Y8k/T61uKLWdgIClo2zYBYpfdYpCOPV6SIMF
+L1dS5M8pac9CkM2NbcEMbZcXykjZmrZidIrDkI3kTPqUjSN+pguUxoJg+I+gUEzQ
+FQj1VJR5SViJk49GMDs0vo+frOAW2kc2RdvvPnuPpplyEySjcMqZtx+vDmVsRAjK
+AFXaPSy5liGCg9j2N9Ab9R6w9JkB3QXM4XFBvQHar6rdP4l/AoIBAQDPkX5IO0T+
+iIgALd+tpyfKywoohzwTlzAM9qsqDmr+BX28Vphu7DOZLhd9zFlKmZHhlj3XXRP0
+Y7Y8JbO+ir2okYNDkA6s7EAl5C/m7RDRvj8d4huPm1rptBq3J0dcwgwOI2Dk1QdC
+CrWEkYLzfObiRUd7gDjBYt9/KsK52Q04iYphs2ItXtrZf6rdFQC+MIJcX35ko9dQ
+cYs+rY3DdwjfEVwwcMTShE/jRMcN4PKnX5QwVzIyZeJPU4DJlNYs3Iaa292GwQ0p
+YnDrYqv0Dy2Tx5/TaEdQdbRX8wGtz+pZresrRMacPN8t8WM7O38Qug5ouwH19DBu
+pjzh5aEUA1i7AoIBABE9aqmKgMCwLL76mS3GOdmSlihsfrGEcbKx8Y+EewJcNKF7
+tNBfHSKwcz53kxzd/wJQ6d1tW2CGeVGKCRc9EU975WUoANHIHUkrtn7zYF4yRx1y
+ssGiktPxiwxRjQV4cxHa0CkzAucexYPbhiMOh/+ac5A1AZObs932z+I+6xYKlUlJ
+8omu4hAvSj36M4QgSnOcU4ASsdj2dFUv5J0aOQ8TwN+H+XmaJ0CIHLa3oca1QSQx
+spT70hqQKLOJVzVMuuYeILh0renOLoleln/ZQsiCjEeu+/IbSZAGOa8V0urNOCFJ
+k9IyMasVP+GUg67PixsMPumSIX79HfISMhoB5TUCggEBAMVULBm/Pvg8FA8XjW4p
+W0sPe7jL1/FH6gZo+pAg5NZZog9Kw9+v7d3SU8LkYn7pQCaWDnSPqEjOApFrxlV+
+0I9QxtmUOl9quhFLvb5r4XGEy7w9GLaNmwBSmJNGZDFqyMsoFxV08FF4nNhK/ZM9
+SsIR2sMuQsaWmKLso/LKxibZmxUG1G8NnkDnfihvryUgOM5YenBy0l9HknkjxYHt
+yCFI/7uNeZAo+Um2OQaYtBcqZlcOjkobUerYF7eMJ5C+lbjjDNbu8PRHAdLFG3QK
+eenj/a2dlS6It8pk21PCNajMDqYz3BzsQcALm6rUBRiByPEH1/VbEDAhGgAnrdq4
+08ECggEBAIO+7DW3vfitCetSYk5oJAVNN+9OMs4SR8/2cbuMc9a/Chc6O/rj58EP
+yH4wSifpnesoZzCah3Ryy+RTYXEwNV65Xopd+J/ANt9MjP8c4j0hY3n/BzpI4aN3
+dpQyXWBbsECZAl0B0LKA2mLMAo72SQYXK9F62Zl6LVoT27wfsADYTqM9GcFagp6/
+fzKQWdLgRps34ysE8PSYkXaIfq/Q0uv1xvDMdW5GpNxcfkikZXQJnuUlxKvdf7Zi
+bdoZvJJ6MPNNoCZBOM+WSitRculiJHfeEMU5VQXpXJby0YEt+Gc4q3Zs1tv3aOqF
+k+WUbp8EAMRqFNRRLcPzcux5vlLvOtw=
+-----END PRIVATE KEY-----
 PEM
 
 my ($pem_fh, $pem_filename) =
